static/js/chart/draw_choropleth.ts

```diff
@@ -38,6 +38,13 @@ import {
   MapPoint,
 } from "./types";
 
+export interface MapZoomParams {
```

Comment:
could you add some documentation for the parameters & interface?

Reply:
done